### PR TITLE
updating readme to chefdk

### PR DESCRIPTION
**Pull request by vinyar**
ok to delete branch when done

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: README.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ delete, and enumerate
...
 
 ## Installation
-Be sure you are running the latest version of Chef, which can be installed
+Be sure you are running the latest version of Chef DK, which can be installed
 via:
 
-    gem install chef
+    https://downloads.chef.io/chef-dk/
 
 This plugin is distributed as a Ruby Gem. To install it, run:
```
Comment (siddheshwar-more) [inline README.md]: I think If you are using chef-dk in that case ruby gems are install using -
`chef gem install <name>`. And also chef-dk comes with inbuilt knife azure, we have to verify how `chef gem install knife-azure` will work here, As recently we found some gem dependency related issues with chef-dk.

Thank you for sharing this.

Comment (siddheshwar-more) [inline README.md]: @vinyar  Please sign CLA.

Comment (andrewelizondo) [inline README.md]: :+1: to the comment about updating the installation command to `chef gem install knife-azure`. I haven't found any dependency conflicts with installing knife-azure into the chef-dk, what conflicts are you referring to @siddheshwar-more?

**Comment by siddheshwar-more** [comment]
@andrewelizondo 
The current `chef-dk` comes with `knife-windows v.0.8` and `chef gem install knife-azure`  installs the `knife-windows v1.1.1` . 
Here's issue https://github.com/chef/chef-dk/issues/411 related to gem dependency. I think we should verify this kind of dependency issues! 

Thanks!

**Comment by siddheshwar-more** [comment]
I think above issue doesn't exists anymore ! Just need to replace  https://github.com/chef/knife-azure/pull/247/files#diff-04c6e90faac2675aa89e2176d2eec7d8R17 by `chef gem install knife-azure`

File: README.md (hunk 1 above)
Comment (vinyar) [inline README.md]: @siddheshwar-more done. Thanks

**Pull request merged**